liquidctl/cli.py

```diff
@@ -279,6 +283,33 @@ def _device_set_color(dev, args, **opts):
     color = map(color_from_str, args['<color>'])
     dev.set_color(args['<channel>'].lower(), args['<mode>'].lower(), color, **opts)
 
+def _device_set_screen(dev, args, **opts):
+    mode = None
+    value = None
+    if args["<mode>"]:
+        mode = args["<mode>"]
+    elif args["brightness"]:
+        mode = "brightness"
+        value = args["<value>"]
+    elif args["orientation"]:
+        mode = "orientation"
+        if args["0"]:
+            value = 0
+        elif args["90"]:
+            value = 90
+        elif args["180"]:
+            value = 180
+        elif args["270"]:
+            value = 270
+    elif args["static"]:
+        mode = "static"
+        value = args["<path>"]
+    elif args["gif"]:
+        mode = "gif"
+        value = args["<path>"]
+    else:
+        raise ValueError()
+    dev.set_screen(mode, value, **opts)
```

Comment:
Most of this is better suited for the driver module, as it's dependent on the device.

Additionally, some things can be simplified.  For example, for the `orientation` mode:

```py
value = args["value"]
if value not in (0, 90, 180, 270):
    raise ValueError("orientation should be 0, 90, 180 or 270 degrees")
```